signature/src/mechanism/cosign/README.md

```diff
@@ -0,0 +1,211 @@
+# Cosign
+
+Cosign is short for Container Signing, Verification and Storage in an OCI registry.
+It aims to make signatures invisible infrastructure.
+[Cosign](https://github.com/sigstore/cosign/blob/main/README.md).
+
+## Policy Format
+
+Cosign is verified due to the container's policy configuration file.
+
+A Policy Requirement of Cosign should be like this
+
+```json
+{
+    "type": "signedBy",
+    "scheme": "cosign",
```

Comment:
Should this document be updated and consistent with the code?

Reply:
Sorry, forgot to update. Fixed and force pushed